getting_started/building.md

```diff
@@ -1,41 +1,59 @@
-# How to build and compile Mbed TLS
+# Building
 
 Mbed TLS supports a number of different build environments out-of-the-box. However, the code and dependencies let you build with any environment.
 
-## UNIX or Cygwin/MinGW with Make
+## Prerequisites
 
-In a regular UNIX environment or Cygwin/MinGW environment, you can build by running:
-    ```
-    make
-    ```
+* GNU Make, CMake or Visual Studio.
+* A C toolchain (compiler, linker, archiver) that supports C99.
+* Python 3.6 to generate the test code.
```

Comment:
Exactly or at least?

Reply:
Good point - at least

Reply:
It was a genuine question, since I know that newer versions may result in some problems with code style checking.